components/ISCNForm.vue

```diff
@@ -277,30 +281,37 @@ const emit = defineEmits<{(e: 'update:modelValue',
   value: ISCNFormData): void
 }>()
 
+const isFormValid = defineModel<boolean>('valid')
```

Comment:
Sorry for changing back-and-forth, but since `defineModel` allow parent to modify this variable, maybe we should just stick with `ref` + `watch` + `emit`

Reply:
[🔥 Clean up form validation logic and emit definition](https://github.com/likecoin/nft-book-press/pull/344/commits/932cbb4321b578e855ed44cf16d833b685e718ff)

Reply:
No need to pass isFormValid from <ISCNForm>